Client/Frontend/Sync/BraveCore/History/Historyv2.swift

```diff
@@ -0,0 +1,221 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Data
+import BraveRewards
+import BraveShared
+import CoreData
+import Shared
+
+private let log = Logger.browserLogger
+
+struct Historyv2Section {
+    var title: String
+    
+    var numberOfObjects: Int
+}
```

Comment:
I don't see this type used anywhere, can be removed?

Reply:
Yes this was part of scraped out implementation.
Fixed at https://github.com/brave/brave-ios/pull/3703/commits/86bbfe1e0a6830bff924eebb8f33ee6518107585